spec/controllers/sign_up/completions_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+describe SignUp::CompletionsController do
+  describe '#show' do
+    it 'tracks page visit' do
+      stub_sign_in
+      subject.session[:sp] = {}
+      stub_analytics
+      allow(@analytics).to receive(:track_event)
+
+      get :show
+
+      expect(@analytics).to have_received(:track_event).with(
+        Analytics::USER_REGISTRATION_AGENCY_HANDOFF_PAGE_VISIT,
+        loa3: nil, service_provider_name: nil
+      )
+    end
```

Comment:
Can we test this with the real expected values? i.e. one scenario with LOA 1 and one with LOA 3.